lib/lor/lib/utils/utils.lua

```diff
@@ -8,6 +8,7 @@ local sgsub = string.gsub
 local smatch = string.match
 local table_insert = table.insert
 local json = require("cjson")
+local st   = require("string.buffer")
```

Comment:
Since the new api `string.buffer` was introduced from OpenResty 1.12.4.1 onwards, it would be nice to have a fallback to be compatible with older versions of OpenResty.